courses/machine_learning/deepdive2/supplemental/solutions/object_detection_with_automl.ipynb

```diff
@@ -0,0 +1,559 @@
+{
```

Comment:
<p>Describe the how the data needs to be formatted directly here instead of pointing to the documentation.</p><p>Here seems to be a good opportunity to explain how the bounding boxes are encoded as well as the label in the dataset.</p><p>You can also add a general explanation about what is object detection, the shape of the training dataset, the notion of bounding boxes, etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>I though the objective here in the notebook was to provide an already prepared image dataset from a public GCS bucket gs://cloud-ml-data/img/openimage/csv/salads_ml_use.csv Or do we want a developer to create their own CSV file with 400+ labeled images for the dataset that the model will use for training?</p>

Reply:
<p>moved the explanation to Step 1</p>